src/components/CommentSection.jsx

```diff
@@ -32,6 +49,12 @@ function CommentSection (){
             <button onClick={commentHandler}>Comments - {reviewComments.length}</button>
             {!showComments ? null :             
             <TableContainer>
+                <form onSubmit={postCommentHandler}>
+                    <br/>
+                    <input type='text' onChange={(event)=>setCommentInput(event.target.value)}/>
```

Comment:
Make this a controlled form - the value property can be set to the commentInput state.